projects/plugins/jetpack/_inc/lib/class.jetpack-search-performance-logger.php

```diff
@@ -77,7 +77,7 @@ public function print_stats() {
 			$encoded_json = '{%22beacons%22:[' . implode(',', $beacons ) . ']}';
 			$encoded_site_url = urlencode( site_url() );
 			$url = "https://pixel.wp.com/boom.gif?v=0.9&u={$encoded_site_url}&json={$encoded_json}";
-			echo '<img src="' . $url . '" width="1" height="1" style="display:none;" alt=":)"/>';
+			echo '<img src="' . esc_html( $url ) . '" width="1" height="1" style="display:none;" alt=""/>';
```

Comment:
Should that be `esc_attr()` instead? Or `esc_url()`?

Reply:
gah, `esc_url` yes. Per the docs: `Always use esc_url when sanitizing URLs (in text nodes, attribute nodes or anywhere else)`

My brain just fell out of my head while writing this one. Thanks for catching it.

Reply:
9fc89b3